packages/components/src/toggle-group-control/test/index.tsx

```diff
@@ -86,31 +89,31 @@ describe( 'ToggleGroupControl', () => {
 			</ToggleGroupControl>
 		);
 
-		const firstRadio = screen.getByRole( 'radio', { name: 'R' } );
-
-		fireEvent.click( firstRadio );
+		await user.click( screen.getByRole( 'radio', { name: 'R' } ) );
 
 		expect( mockOnChange ).toHaveBeenCalledWith( 'rigas' );
 	} );
-	it( 'should render tooltip where `showTooltip` === `true`', () => {
+	it( 'should render tooltip where `showTooltip` === `true`', async () => {
+		const user = userEvent.setup( {
+			advanceTimers: jest.advanceTimersByTime,
+		} );
 		render(
 			<ToggleGroupControl label="Test Toggle Group Control">
 				{ optionsWithTooltip }
 			</ToggleGroupControl>
 		);
 
-		const firstRadio = screen.getByLabelText(
-			'Click for Delicious Gnocchi'
-		);
-
-		fireEvent.focus( firstRadio );
+		await user.tab();
```

Comment:
It's not immediately clear what this does. Should we keep the `firstRadio` and then just do a `firstRadio.focus()`?

Reply:
Addressed via https://github.com/WordPress/gutenberg/pull/45627#discussion_r1018074252